yt_dlp/extractor/youtube.py

```diff
@@ -6965,22 +6965,30 @@ class YoutubeSearchIE(YoutubeTabBaseInfoExtractor, SearchInfoExtractor):
     IE_DESC = 'YouTube search'
     IE_NAME = 'youtube:search'
     _SEARCH_KEY = 'ytsearch'
-    _SEARCH_PARAMS = 'EgIQAQ%3D%3D'  # Videos only
+    _SEARCH_PARAMS = 'EgIIAfABAQ=='  # Videos only
     _TESTS = [{
         'url': 'ytsearch5:youtube-dl test video',
         'playlist_count': 5,
```

Comment:
the new params yield only 2 results, while the old params still yield 5

Reply:
my bad. Seems I copied the wrong params. What we want is https://github.com/iv-org/invidious/blame/99a5e9cbc44daa4555f36b43bc4b7246aee454c9/spec/invidious/search/yt_filters_spec.cr#L23